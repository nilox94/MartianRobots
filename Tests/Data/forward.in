3 3
1 1 N
F
1 1 E
F
1 1 S
F
1 1 W
F
