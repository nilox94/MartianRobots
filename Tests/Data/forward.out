1 2 N
2 1 E
1 0 S
0 1 W
